docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plt.loglog(x,y_means,label=r'$\hat{T}_\epsilon$',c=c1,linewidth=1.5)
</pre><p>Label is added, but no legend shown, think <code>plt.legend()</code> should fix this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>